src/chromeDebugAdapter.ts

```diff
@@ -147,8 +149,9 @@ export class ChromeDebugAdapter extends CoreDebugAdapter {
                 });
             }
 
+            const userDataDir = (args.userDataDir || undefined);
```

Comment:
In which scenario do we use this undefined?

Reply:
There is a case where it can still be undefined (i.e. when userDataDir is not passed, and runtimeExecutable is not set). At this point, we know that it will be either a string or undefined (not a boolean).

Reply:
I'm confused as to how we use undefined then. If args.userDataDir is undefined then const userDataDir = args.userDataDir; will have the same effect as this line.

Reply:
Changed with refactoring so this shouldn't be an issue anymore